ci-operator/step-registry/telcov10n/functional/cnf-ran/ibu-deploy-spoke-master-vms/telcov10n-functional-cnf-ran-ibu-deploy-spoke-master-vms-commands.sh

```
@@ -0,0 +1,73 @@
+      echo "$(basename "${filename}")": \'"$(cat "$filename")"\'
+    fi
+  done > "${dest_file}"
+
+  echo "Processing complete. Check \"${dest_file}\""
+}
+
+echo "Create group_vars directory"
+mkdir -p /eco-ci-cd/inventories/ocp-deployment/group_vars
+
+echo "Copy group inventory files from SHARED_DIR"
+cp ${SHARED_DIR}/all /eco-ci-cd/inventories/ocp-deployment/group_vars/all
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)